dev/io.openliberty.microprofile.health.internal.common/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -18,5 +18,8 @@
 mpHealth=MicroProfile Health
 mpHealth.desc=Configuration for MicroProfile Health.
 
-fileUpdateInterval=Interval of time at which to update the file-based health checks.
-fileUpdateInterval.desc=The value that you specify sets the interval at which the health checks are queried for the file-based health checks. The value is a number followed by an optional time unit of ms for milliseconds or s for seconds. If no time unit is specified, the value is in seconds by default.
\ No newline at end of file
+checkInterval=Interval of time at which to update the file-based health checks.
+checkInterval.desc=The value that you specify sets the interval at which the health checks are queried for the file-based health checks. The value is a number followed by an optional time unit of ms for milliseconds or s for seconds. If no time unit is specified, the value is in seconds by default.
```

Comment:
Is there a default value for `checkInterval`?

Reply:
There is no default value for `checkInterval` ormerly `fileUpdateInterval`. The existence/configuration of it enables functionality.

Reply:
Might want to state that if its empty value, the functionality is disabled?